test/createComponent.spec.js

```diff
@@ -26,8 +36,14 @@ describe('createComponent', function () {
         componentDidMount: sinon.match.func,
         componentWillUnmount: sinon.match.func
       });
-    expect(myComponentInstance).to.be.instanceof(MyComponent);
+  });
+
+  it('is a valid React component and a MyComponent\'s instance', function () {
     expect('isReactComponent' in Object.getPrototypeOf(myComponentInstance)).to.be.equal(true);
+    expect(myComponentInstance).to.be.instanceof(MyComponent);
+  });
+
+  it('has the spec\'s functions', function () {
```

Comment:
This is not an actual review comment, just a Javascript question: isn't 
```
"has the spec's functions"
```
nicer? Or the linting doesn't allow double quotes at all?